examples/virtual-field/README.md

```diff
@@ -0,0 +1,121 @@
+## Feature Example - virtual field
+
+This project implements a basic **Blog**, with `Posts` and `Authors`.
+
+You can use it as a starting place for learning how to use Keystone.
+
+## Instructions
+
+To run this project, clone the Keystone repository locally then navigate to this directory and run:
+
+```shell
+yarn dev
+```
+
+This will start the Admin UI at [localhost:3000](http://localhost:3000).
+You can use the Admin UI to create items in your database.
+
+You can also access a GraphQL Playground at [localhost:3000/api/graphql](http://localhost:3000/api/graphql), which allows you to directly run GraphQL queries and mutations.
+
+🚀 Congratulations, you're now up and running with Keystone!
+
+## Features
+
+This project demonstrates how to use virtual fields.
+
```

Comment:
It uses the `schema` API from the `@keystone-next/types` package to define the GraphQL schema used by the virtual fields.

Reply:
Gah, sorry, meant to be an additional sentence in the opening paragraph.